linux_os/guide/system/bootloader-zipl/zipl_audit_argument/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Enable Auditing to Start Prior to the Audit Daemon in zIPL'
+
+description: |-
+    To ensure all processes can be audited, even those which start prior to the audit daemon,
+    check that all boot entries in <tt>/boot/loader/entries/*.conf</tt> have <tt>audit=1</tt>
+    included in its options.
+    Make sure <tt>/etc/zipl.conf</tt> doesn't contain <pre>image = </pre> setting,
+    as {{{ full_name }}} adheres to Boot Loader Specification (BLS),
+    And run <pre>zipl</pre> command so that <tt>/boot/bootmap</tt> is updated.
+
+    To ensure that new kernels and boot entries continue to enable audit,
+    add <pre>audit=1</pre> to <tt>/etc/kernel/cmdline</tt>.
+
+rationale: |-
+    Each process on the system carries an "auditable" flag which indicates whether
+    its activities can be audited. Although <tt>auditd</tt> takes care of enabling
+    this for all processes which launch after it does, adding the kernel argument
+    ensures it is set for every process during boot.
+
+severity: medium
+
+ocil_clause: 'auditing is not enabled at boot time'
+
+ocil: |-
+  To check that all boot entries enable audit at boot time, run the following command:
+  <pre>sudo grep -L "\baudit=1\b" /boot/loader/entries/*.conf</pre>
```

Comment:
Does it makes sense to have ^options.* there as well? To be really sure we are checking the right line.